src/openms/source/ANALYSIS/OPENSWATH/TargetedSpectraExtractor.cpp

```diff
@@ -266,21 +276,18 @@ namespace OpenMS
           // iterate through the subordinate level
           for (const auto& subordinate : feature.getSubordinates())
           {
-            construct_feature(subordinate);
+            construct_feature(subordinate, mz_tolerance_, rt_window_);
           }
         }
         else
         {
-          construct_feature(feature);
+          construct_feature(feature, mz_tolerance_, rt_window_);
         }
       }
     }
   }
 
-  void TargetedSpectraExtractor::searchSpectrum(
-      OpenMS::FeatureMap& feat_map,
-      OpenMS::FeatureMap& feat_map_output,
-      bool add_unidentified_features) const
+  void TargetedSpectraExtractor::searchSpectrum(OpenMS::FeatureMap& feat_map, OpenMS::FeatureMap& feat_map_output, bool add_unidentified_features) const
```

Comment:
Can't see it from my phone but this change looks like the line could be too long now.